DEBUG 2015-06-11 18:17:37,304: 	 Executing command: 'kill'
DEBUG 2015-06-11 18:17:37,305: 	 Found CRAB cache file /afs/cern.ch/user/c/cheidegg/.crab3
ERROR 2015-06-11 18:17:37,306: 	 Error: Please indicate the CRAB project directory with --dir=<project-directory>.
ERROR 2015-06-11 18:17:37,306: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/bin/crab", line 136, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/bin/crab", line 122, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/lib/python2.6/site-packages/CRABClient/Commands/SubCommand.py", line 253, in __init__
    self.validateOptions()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/lib/python2.6/site-packages/CRABClient/Commands/kill.py", line 51, in validateOptions
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/lib/python2.6/site-packages/CRABClient/Commands/SubCommand.py", line 559, in validateOptions
    raise ex
MissingOptionException: [91mError[0m: Please indicate the CRAB project directory with --dir=<project-directory>.
DEBUG 2015-06-11 18:17:50,231: 	 Executing command: 'kill'
DEBUG 2015-06-11 18:17:50,231: 	 Found CRAB cache file /afs/cern.ch/user/c/cheidegg/.crab3
DEBUG 2015-06-11 18:17:50,249: 	 Checking credentials
DEBUG 2015-06-11 18:17:50,250: 	 Found proxy file /tmp/x509up_u66803
DEBUG 2015-06-11 18:17:50,250: 	 Getting proxy life time left
DEBUG 2015-06-11 18:17:50,855: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u66803 -timeleft
 output : 85988

 error: 
 retcode : 0
DEBUG 2015-06-11 18:17:51,566: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u66803 -actimeleft
 output : 85987

 error: 
 retcode : 0
DEBUG 2015-06-11 18:17:51,566: 	 Proxy valid for 23:53:08 hours
ERROR 2015-06-11 18:17:51,566: 	 Cannot find .requestcache file in CRAB project directory /afs/cern.ch/user/c/cheidegg/p/CMSSW_7_4_4_patch2/src/MetScanning/skim/crab/2015-06-12_runa_flat
ERROR 2015-06-11 18:17:51,567: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/bin/crab", line 136, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/bin/crab", line 122, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/lib/python2.6/site-packages/CRABClient/Commands/SubCommand.py", line 317, in __init__
    self.loadLocalCache(proxyOptsSetPlace)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/lib/python2.6/site-packages/CRABClient/Commands/SubCommand.py", line 431, in loadLocalCache
    self.cachedinfo, self.logfile = loadCache(self.requestarea, self.logger)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/lib/python2.6/site-packages/CRABClient/ClientUtilities.py", line 412, in loadCache
    raise CachefileNotFoundException(msg)
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/cern.ch/user/c/cheidegg/p/CMSSW_7_4_4_patch2/src/MetScanning/skim/crab/2015-06-12_runa_flat
DEBUG 2015-06-11 18:19:12,986: 	 Executing command: 'kill'
DEBUG 2015-06-11 18:19:12,986: 	 Found CRAB cache file /afs/cern.ch/user/c/cheidegg/.crab3
ERROR 2015-06-11 18:19:12,987: 	 Error: 2015-06-12_runa_flat/crab_ZeroBias*_Run2015A-PromptReco-v1_RECO/ is not a valid CRAB project directory.
ERROR 2015-06-11 18:19:12,988: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/bin/crab", line 136, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/bin/crab", line 122, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/lib/python2.6/site-packages/CRABClient/Commands/SubCommand.py", line 253, in __init__
    self.validateOptions()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/lib/python2.6/site-packages/CRABClient/Commands/kill.py", line 51, in validateOptions
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc481/cms/crabclient/3.3.1506/lib/python2.6/site-packages/CRABClient/Commands/SubCommand.py", line 563, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: 2015-06-12_runa_flat/crab_ZeroBias*_Run2015A-PromptReco-v1_RECO/ is not a valid CRAB project directory.
